teamB/kimSungHoon/controllers/postController.js

```diff
@@ -0,0 +1,65 @@
+const postService = require('../services/postService');
+
+const createPost = async (req, res) => {
+    try {
+        const {title, content, user_id} = req.body;
+
+        if( !title || !content || !user_id ){   
+            return res.status(400).json({message: '키 에러 발생'});
+        }
+
+        await postService.createPost(title, content, user_id);
+        return res.status(201).json({message: '게시글 등록 성공'});
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+};
+
+const readingPost = async (req, res) => {
+    try {
+        const result = await postService.readingPost();
+        return res.status(201).json({ data: result });
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+};
+
+const updatePost = async (req, res) => {
+    try {
+        const {content} = req.body;
+        const postId = req.params.postId;
+
+        if( !content ){   
+            return res.status(400).json({message: '키 에러 발생'});
+        }
+        const data = await postService.updatePost(content, postId);
+        return res.status(201).json(data)
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+}
+
+const deletePost = async (req, res) => {
+    try {
+        const postId = req.params.postId;
+        const result = await postService.deletePost(postId);
+        return res.status(204);
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+}
+
+module.exports = {
+	    createPost,
+    	readingPost,
+    	updatePost,
+    	deletePost
```

Comment:
indentation 맞춰서 수정해주세요 !
```suggestion
	createPost,
    	readingPost,
    	updatePost,
    	deletePost
```

Reply:
vscode 상에서는 정상적으로 되어있어서 왜 이런지 모르겠습니다.. 일단 기존 vscode 에서 createPost는 안쪽으로 들여쓰기 하겠습니다.